iaso/api/entity_types.py

```diff
@@ -88,6 +89,11 @@ def create(self, request, pk=None):
 
         if not request.user.has_perm(permission.ENTITY_TYPE_WRITE):
             return Response(status=status.HTTP_403_FORBIDDEN)
+        name = request.data.get("name", None)
+        account = request.data.get("account", None)
+        if name is not None and account is not None:
+            name_to_code = name.replace(" ", "_")
+            request.data["code"] = f"{name_to_code.lower()}_{account}"
```

Comment:
Just to go all the way to make it a "code", I propose we first sanitize the name to remove special chars.
For example with `unidecode` (already in the project):

```python
from unidecode import unidecode

name = "Bén Härper"

unidecode(name).lower().replace(" ", "_")
# OUTPUT: 'ben_harper'
```